tools/spectral/ipa/rulesets/IPA-104.yaml

```
@@ -21,3 +22,11 @@ rules:
     given: '$.paths[*].get'
     then:
       function: 'getMethodReturnsSingleResource'
+
+  xgen-IPA-104-get-method-response-code-is-200-OK:
```

```suggestion
  xgen-IPA-104-get-method-response-code-is-200:
```
[Nit] If we want it a bit shorter, I think just "200" is enough